index.html

```diff
@@ -195,6 +195,20 @@ <h2>About <br /> Myself</h2>
         </div>
     </section>
 
+    <footer class="footer" id="contact">
+        <h2>
+            Contact Me
+        </h2>
+        <p> If you have an application you are interested in developing, a feature that you need built or a project that
+            needs coding. I’d love to help with it </p>
+        <form class="contact__form" method="post" action="https://formspree.io/f/xeqngaoy">
+            <input type="text" maxlength="30" name="full_name" placeholder="Full Name" required>
+            <input type="email" name="email" placeholder="Email Adress" required>
+            <textarea name="msg" placeholder="Write your message here " maxlength="500" required></textarea>
+            <input type="submit" value="Get Tt Touch">
```

Comment:
- [ ] Please, kindly fix the typo "Get Tt Touch" 👍 
- [ ] Please, consider the button interactions such as hover, active (when it's pressed), disabled, and enable (default state) are styled as shown in the Figma template:
<img width="837" alt="Screen Shot 2022-02-16 at 01 26 39" src="https://user-images.githubusercontent.com/49064106/154152114-1bba61f3-f92a-4aad-9b13-c68097fa50ce.png">